python/py-tensorboard/Portfile

```diff
@@ -74,8 +74,4 @@ if {${name} ne ${subport}} {
         --root ${destroot} \
         ${distpath}/${distfiles}
     destroot.post_args
-
-    livecheck.type      none
```

Comment:
Please put this back. It is intentional that the version specific ports have livecheck none.

Reply:
I don't agree with this, but I'll back it out. We can decide as a group on macports-dev@.

Reply:
I am find with a open discussion on this, but yes for now please out them back. I for one like the current behaviour as it works well with how I personally check the ports I maintain for updates.